README.md

```diff
@@ -9,6 +9,14 @@ This will prevent you from using "soft" dependencies that are not defined within
 [![Dependency Status](https://www.versioneye.com/user/projects/565df3b9b6f5ff00380001ea/badge.svg?style=flat)](https://www.versioneye.com/user/projects/565df3b9b6f5ff00380001ea)
 [![Code Coverage](https://scrutinizer-ci.com/g/maglnet/ComposerRequireChecker/badges/coverage.png?b=master)](https://scrutinizer-ci.com/g/maglnet/ComposerRequireChecker/?branch=master)
 
+## What's it about?
+
+Your code most certainly uses external dependencies. And if it's only extending PHPUnits `TestCase`. And to make sure that your code knows where to find that `TestCase`-class you for sure called `composer require --dev phpunit/phpunit`. That means your dependency is a hard dependency. 
```

Comment:
Sorry, I can't understand it... Looks like you are trying to describe what is hard dependency by providing an example of it, but grammar here needs to be fixed

Reply:
What exactly is your issue? What do you not understand? For me that's valid english grammar.

Reply:
How would you change it?

Reply:
> Your code most certainly uses external dependencies. And if it's only extending PHPUnits `TestCase`. And...

`IF sth THEN sth`
not
`AND IF sth. AND DO sth.`

Reply:
It's called [Polysyndeton](https://en.wikipedia.org/wiki/Polysyndeton) and should intensify the causality of the matter.

So still the question is how you would want to change it. As I said in the initial comment: 

> Not sure it's perfect, but it should be a starting point… 😉